src/constants/remoteConfigConstants.js

```diff
@@ -51,6 +52,7 @@ export const INITIAL_REMOTE_CONFIG = {
   [REMOTE_CONFIG.FEATURE_AAVE]: true,
   [REMOTE_CONFIG.FEATURE_POOL_TOGETHER]: true,
   [REMOTE_CONFIG.FEATURE_ALTALIX]: true,
+  [REMOTE_CONFIG.FEATURE_STORIES]: true,
```

Comment:
After a chat with the people making the content, could we please turn this off for now? Thanks!